.github/workflows/operator-e2e-integration-tests.yml

```diff
@@ -55,6 +55,12 @@ jobs:
           # Run the e2e tests
           cd infra/feast-operator/
           make test-e2e
+          docker images --format '{{.Repository}}:{{.Tag}}' | grep 'feast' | xargs -r docker rmi -f
```

Comment:
may be you can add it as separate step for more clarity in case if there is a failure? Even if it makes sense to have its own github action?

Reply:
> may be you can add it as separate step for more clarity in case if there is a failure? Even if it makes sense to have its own github action?

I had to do it here otherwise the tests were failing because of out of space error(Github allocated size is only 14gb and can't be changed or affected)

Reply:
> I might be missing something, but could you clarify how the compute/memory can change if it's run in a separate step within the same action?

Maybe I misunderstood what you mean, you want me to split all the lines after `make test-e2e` into different steps, I can do that no problem there

Reply:
Sorry for the confusion from my original message:

1) Moving it to a separate step in the same action.
2) Moving it to the its own github action. This is helpful especially for the compute/memory reasons so that you no need to share the resources with the existing tests.

#1 makes more sense to do so that easy to identify the failures and logs are also separated into its own step.
#2 you can decide based on pros and cons of each.

Reply:
> Sorry for the confusion from my original message:
> 
> 1. Moving it to a separate step in the same action.
> 2. Moving it to the its own github action. This is helpful especially for the compute/memory reasons so that you no need to share the resources with the existing tests.
> 
> #1 makes more sense to do so that easy to identify the failures and logs are also separated into its own step. #2 you can decide based on pros and cons of each.

@lokeshrangineni I chose option 1 and updated the PR, please have a look